packages/muelu/research/regionMG/src/SetupRegionVector_def.hpp

```diff
@@ -305,19 +268,16 @@ void regionalToComposite(const Array<RCP<Xpetra::MultiVector<Scalar, LocalOrdina
  *  composite layout takes care of the summation of interface values.
  */
 template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
-void sumInterfaceValues(Array<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> > >& regVec,
-                        const std::vector<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > > revisedRowMapPerGrp,///< revised row maps in region layout [in]
-                        const std::vector<RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > > rowImportPerGrp ///< row importer in region layout [in])
+void sumInterfaceValues(RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> >& regVec,
+                        const RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> >  revisedRowMapPerGrp,///< revised row maps in region layout [in]
+                        const RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> >  rowImportPerGrp ///< row importer in region layout [in])
```

Comment:
The removed `std::vector` was related to the concept of "groups". Hence, this could be renamed to 
```suggestion
                        const RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> >  revisedRowMap,///< revised row maps in region layout [in]
                        const RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> >  rowImport ///< row importer in region layout [in])
```